music_assistant/server/helpers/api.py

```diff
@@ -93,12 +93,12 @@ def parse_value(name: str, value: Any, value_type: Any, default: Any = MISSING)
     if value is None and value_type is NoneType:
         return None
     origin = get_origin(value_type)
-    if origin is list:
-        return [
+    if origin in (tuple, list):
+        return (
```

Comment:
```suggestion
        return origin(
```
this should make a tuple become a tuple again and a list a list